### PR TITLE
Bump gtk

### PR DESCRIPTION
**Pull request by JAicewizard**
Build time hasn't changed:

before:
`Finished dev [unoptimized + debuginfo] target(s) in 1m 02s`
after:
`Finished dev [unoptimized + debuginfo] target(s) in 1m 06s`

There are a couple less dependencies, but those didn't seem to have any effect on total build times. This also removed direct dependencies from druid-shell. This will reduce maintenance effort a bit.

### CODE REVIEW

File: druid-shell/Cargo.toml [hunk 1]
```diff
@@ -15,7 +15,7 @@ default-target = "x86_64-pc-windows-msvc"
 
 [features]
 default = ["gtk"]
-gtk = ["gio", "gdk", "gdk-sys", "glib", "glib-sys", "gtk-sys", "gtk-rs", "gdk-pixbuf"]
+gtk = ["gdk-sys", "glib-sys", "gtk-sys", "gtk-rs"]
```
Comment (bilelmoussaoui) [inline druid-shell/Cargo.toml:18]: You don't need -sys neither, they are re-exported as ffi.

**Review by PoignardAzur — APPROVED** [review]
LGTM.

Rebase and merge?

**Comment by JAicewizard** [comment]
Blocked on piet version bump.

**Comment by cmyr** [comment]
@JAicewizard I've released piet `0.5.0-pre1`, which you can use to test this out. Let me know if you have any issues?

**Comment by JAicewizard** [comment]
rebased, and it seems to be working.

**Comment by JAicewizard** [comment]
rebased, CI seems to have issues but it compiles fine locally.

File: druid-shell/Cargo.toml [hunk 2]
```diff
@@ -15,8 +15,8 @@ default-target = "x86_64-pc-windows-msvc"
 
 [features]
 default = ["gtk"]
-gtk = ["gio", "gdk", "gdk-sys", "glib", "glib-sys", "gtk-sys", "gtk-rs", "gdk-pixbuf"]
-x11 = ["x11rb", "nix", "cairo-sys-rs", "bindgen", "pkg-config"]
+gtk = ["gdk-sys", "glib-sys", "gtk-sys", "gtk-rs"]
+x11 = ["x11rb", "nix", "cairo-sys-rs"]
```
Comment (Maan2003) [inline druid-shell/Cargo.toml:19]: bindgen and pkg-config got removed, maybe in rebasing/merging somewhere?

**Pull request merged**